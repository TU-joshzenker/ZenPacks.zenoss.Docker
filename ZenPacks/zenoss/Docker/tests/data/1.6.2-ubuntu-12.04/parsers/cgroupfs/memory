
/sys/fs/cgroup/memory/

/sys/fs/cgroup/memory/docker

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.kmem.tcp.max_usage_in_bytes
0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.kmem.tcp.failcnt
0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.kmem.tcp.usage_in_bytes
0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.kmem.tcp.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.kmem.slabinfo

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.kmem.max_usage_in_bytes
0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.kmem.failcnt
0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.kmem.usage_in_bytes
0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.kmem.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.numa_stat
total=1940 N0=1940
file=1587 N0=1587
anon=353 N0=353
unevictable=0 N0=0
hierarchical_total=1940 N0=1940
hierarchical_file=1587 N0=1587
hierarchical_anon=353 N0=353
hierarchical_unevictable=0 N0=0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.pressure_level

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.oom_control
oom_kill_disable 0
under_oom 0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.move_charge_at_immigrate
0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.swappiness
60

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.use_hierarchy
0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.force_empty

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.stat
cache 6541312
rss 1404928
rss_huge 0
mapped_file 2281472
writeback 0
pgpgin 2324
pgpgout 384
pgfault 2766
pgmajfault 47
inactive_anon 4096
active_anon 1441792
inactive_file 6270976
active_file 229376
unevictable 0
hierarchical_memory_limit 18446744073709551615
total_cache 6541312
total_rss 1404928
total_rss_huge 0
total_mapped_file 2281472
total_writeback 0
total_pgpgin 2324
total_pgpgout 384
total_pgfault 2766
total_pgmajfault 47
total_inactive_anon 4096
total_active_anon 1441792
total_inactive_file 6270976
total_active_file 229376
total_unevictable 0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.failcnt
0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.soft_limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.max_usage_in_bytes
8056832

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/memory.usage_in_bytes
7946240

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/notify_on_release
0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/tasks
1648
1667

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cgroup.clone_children
0

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cgroup.event_control

/sys/fs/cgroup/memory/docker/8527dc5f078078839dc0e9dbfeed68fda8e90daf83d7f6c17a7be64a83cc63fa/cgroup.procs
1648
1667

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.kmem.tcp.max_usage_in_bytes
0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.kmem.tcp.failcnt
0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.kmem.tcp.usage_in_bytes
0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.kmem.tcp.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.kmem.slabinfo

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.kmem.max_usage_in_bytes
0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.kmem.failcnt
0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.kmem.usage_in_bytes
0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.kmem.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.numa_stat
total=22232 N0=22232
file=549 N0=549
anon=21683 N0=21683
unevictable=0 N0=0
hierarchical_total=22232 N0=22232
hierarchical_file=549 N0=549
hierarchical_anon=21683 N0=21683
hierarchical_unevictable=0 N0=0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.pressure_level

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.oom_control
oom_kill_disable 0
under_oom 0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.move_charge_at_immigrate
0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.swappiness
60

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.use_hierarchy
0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.force_empty

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.stat
cache 2265088
rss 93753344
rss_huge 0
mapped_file 516096
writeback 0
pgpgin 439868665
pgpgout 439845223
pgfault 439869849
pgmajfault 17
inactive_anon 0
active_anon 93769728
inactive_file 2187264
active_file 61440
unevictable 0
hierarchical_memory_limit 18446744073709551615
total_cache 2265088
total_rss 93753344
total_rss_huge 0
total_mapped_file 516096
total_writeback 0
total_pgpgin 439868665
total_pgpgout 439845223
total_pgfault 439869849
total_pgmajfault 17
total_inactive_anon 0
total_active_anon 93769728
total_inactive_file 2187264
total_active_file 61440
total_unevictable 0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.failcnt
0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.soft_limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.max_usage_in_bytes
271425536

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/memory.usage_in_bytes
99328000

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/notify_on_release
0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/tasks
1587
1611
1612
1613
1614
1615
1616
1617
1618
1619
1620
1621
1622
1623
1624

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cgroup.clone_children
0

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cgroup.event_control

/sys/fs/cgroup/memory/docker/9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f/cgroup.procs
1587
1611
1612
1613
1614
1615
1616
1617
1618
1619
1620
1621
1622
1623
1624

/sys/fs/cgroup/memory/docker/memory.kmem.tcp.max_usage_in_bytes
0

/sys/fs/cgroup/memory/docker/memory.kmem.tcp.failcnt
0

/sys/fs/cgroup/memory/docker/memory.kmem.tcp.usage_in_bytes
0

/sys/fs/cgroup/memory/docker/memory.kmem.tcp.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/memory.kmem.slabinfo

/sys/fs/cgroup/memory/docker/memory.kmem.max_usage_in_bytes
0

/sys/fs/cgroup/memory/docker/memory.kmem.failcnt
0

/sys/fs/cgroup/memory/docker/memory.kmem.usage_in_bytes
0

/sys/fs/cgroup/memory/docker/memory.kmem.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/memory.numa_stat
total=0 N0=0
file=0 N0=0
anon=0 N0=0
unevictable=0 N0=0
hierarchical_total=0 N0=0
hierarchical_file=0 N0=0
hierarchical_anon=0 N0=0
hierarchical_unevictable=0 N0=0

/sys/fs/cgroup/memory/docker/memory.pressure_level

/sys/fs/cgroup/memory/docker/memory.oom_control
oom_kill_disable 0
under_oom 0

/sys/fs/cgroup/memory/docker/memory.move_charge_at_immigrate
0

/sys/fs/cgroup/memory/docker/memory.swappiness
60

/sys/fs/cgroup/memory/docker/memory.use_hierarchy
0

/sys/fs/cgroup/memory/docker/memory.force_empty

/sys/fs/cgroup/memory/docker/memory.stat
cache 0
rss 0
rss_huge 0
mapped_file 0
writeback 0
pgpgin 0
pgpgout 0
pgfault 0
pgmajfault 0
inactive_anon 0
active_anon 0
inactive_file 0
active_file 0
unevictable 0
hierarchical_memory_limit 18446744073709551615
total_cache 0
total_rss 0
total_rss_huge 0
total_mapped_file 0
total_writeback 0
total_pgpgin 0
total_pgpgout 0
total_pgfault 0
total_pgmajfault 0
total_inactive_anon 0
total_active_anon 0
total_inactive_file 0
total_active_file 0
total_unevictable 0

/sys/fs/cgroup/memory/docker/memory.failcnt
0

/sys/fs/cgroup/memory/docker/memory.soft_limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/memory.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/docker/memory.max_usage_in_bytes
0

/sys/fs/cgroup/memory/docker/memory.usage_in_bytes
0

/sys/fs/cgroup/memory/docker/notify_on_release
0

/sys/fs/cgroup/memory/docker/tasks

/sys/fs/cgroup/memory/docker/cgroup.clone_children
0

/sys/fs/cgroup/memory/docker/cgroup.event_control

/sys/fs/cgroup/memory/docker/cgroup.procs

/sys/fs/cgroup/memory/memory.kmem.tcp.max_usage_in_bytes
0

/sys/fs/cgroup/memory/memory.kmem.tcp.failcnt
0

/sys/fs/cgroup/memory/memory.kmem.tcp.usage_in_bytes
0

/sys/fs/cgroup/memory/memory.kmem.tcp.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/memory.kmem.slabinfo

/sys/fs/cgroup/memory/memory.kmem.max_usage_in_bytes
0

/sys/fs/cgroup/memory/memory.kmem.failcnt
0

/sys/fs/cgroup/memory/memory.kmem.usage_in_bytes
0

/sys/fs/cgroup/memory/memory.kmem.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/memory.numa_stat
total=34547 N0=34547
file=25944 N0=25944
anon=8603 N0=8603
unevictable=0 N0=0
hierarchical_total=67472 N0=67472
hierarchical_file=28080 N0=28080
hierarchical_anon=39392 N0=39392
hierarchical_unevictable=0 N0=0

/sys/fs/cgroup/memory/memory.pressure_level

/sys/fs/cgroup/memory/memory.oom_control
oom_kill_disable 0
under_oom 0

/sys/fs/cgroup/memory/memory.move_charge_at_immigrate
0

/sys/fs/cgroup/memory/memory.swappiness
60

/sys/fs/cgroup/memory/memory.use_hierarchy
0

/sys/fs/cgroup/memory/memory.force_empty

/sys/fs/cgroup/memory/memory.stat
cache 106586112
rss 35004416
rss_huge 0
mapped_file 15540224
writeback 0
pgpgin 266253
pgpgout 231685
pgfault 1146681
pgmajfault 519
inactive_anon 192512
active_anon 35041280
inactive_file 73498624
active_file 32845824
unevictable 0
hierarchical_memory_limit 18446744073709551615
total_cache 115392512
total_rss 166367232
total_rss_huge 0
total_mapped_file 18337792
total_writeback 4096
total_pgpgin 440146081
total_pgpgout 440077292
total_pgfault 441028136
total_pgmajfault 583
total_inactive_anon 196608
total_active_anon 166457344
total_inactive_file 81956864
total_active_file 33136640
total_unevictable 0

/sys/fs/cgroup/memory/memory.failcnt
0

/sys/fs/cgroup/memory/memory.soft_limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/memory.limit_in_bytes
18446744073709551615

/sys/fs/cgroup/memory/memory.max_usage_in_bytes
0

/sys/fs/cgroup/memory/memory.usage_in_bytes
287481856

/sys/fs/cgroup/memory/release_agent


/sys/fs/cgroup/memory/notify_on_release
0

/sys/fs/cgroup/memory/tasks
1
2
3
5
7
8
9
10
11
12
13
14
15
16
17
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
37
38
39
40
41
42
43
55
57
76
77
205
206
360
364
371
380
387
429
430
522
594
811
825
863
874
875
876
901
908
918
919
922
933
934
942
969
972
973
990
1090
1091
1195
1199
1200
1201
1202
1204
1206
1260
1360
1361
1465
1469
1470
1544
1557
1561
1574
1575
1584
1625
1629
1636
1637
1638
1639
1643
1644
1645
1646
1647
1653
1666
2414
3113
3620
3917
3950
4011
4015
4125

/sys/fs/cgroup/memory/cgroup.sane_behavior
0

/sys/fs/cgroup/memory/cgroup.clone_children
0

/sys/fs/cgroup/memory/cgroup.event_control

/sys/fs/cgroup/memory/cgroup.procs
1
2
3
5
7
8
9
10
11
12
13
14
15
16
17
19
20
21
22
23
24
25
26
27
28
29
30
31
32
33
34
37
38
39
40
41
42
43
55
57
76
77
205
206
360
364
371
380
387
429
430
522
594
811
825
863
901
908
918
919
922
933
934
942
969
972
990
1090
1091
1195
1199
1260
1360
1361
1465
1469
1470
1575
1584
1636
1643
2414
3113
3620
3950
4011
4015
4129
