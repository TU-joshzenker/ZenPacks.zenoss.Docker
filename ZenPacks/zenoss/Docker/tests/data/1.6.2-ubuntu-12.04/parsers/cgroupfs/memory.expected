# vim: ft=yaml
command: '/usr/bin/env sudo find /sys/fs/cgroup/memory/ -printf "\n%p\n" -exec cat {} 2>/dev/null \;'

values:
  9ffff877e5c702dde9583ff8f0d5cdc2a01bbb26fef7ac23e7140fd6f88fb69f:
    usage: 99328000
